### PR TITLE
[WIP] ALICE GTP proposals for GSoC 2020

### PR DESCRIPTION
**Pull request by Atlantic777**
We have to Grid related project proposals to add. cc @costing

### CODE REVIEW

File: _gsocproposals/2020/proposal_ALICEgridcontainers.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+layout: gsoc_proposal
+project: ALICE
+year: 2020
+organization:
+  - CERN
+---
+
+## Description
+
+The ALICE Experiment at the CERN LHC relies on the WLCG, Worldwide LHC Computing Grid, for processing and analysing large quantities of its data. With clusters spread across hundreds computing sites worldwide, most of the computing resources provided by these sites vary greatly in both hardware and configurations. To assure a compatibility with the software requirements of its applications, ALICE is moving towards a new framework for executing tasks on the Grid using containers. To execute containers, the sites must  meet a series of prerequisites both in terms of operating system and container handling software. The readiness of the site to run containers must be ensured before configuring the task submission using the ALICE container framework. To that end, a test framework and testing methodology must be developed and executed on all sites providing resources for ALICE. The results of the test must be analysed and reported back to the site administrators for action in case of issues or incomplete provision of the necessary prerequisites.
```
Comment (agheata) [inline _gsocproposals/2020/proposal_ALICEgridcontainers.md]: please add inline links to acronyms like LHC or WLCG

File: _gsocproposals/2020/proposal_ALICEgridcontainers.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## Task ideas
+
+ * Develop a simple submission framework within the JAliEn Grid environment for pre- configured containers to specific Grid sites;
+ * Use the framework to start containers, gather and analyze the output of the submission to gauge the readiness of the site to execute the containers against a list of known conditions; 
+ * Suggest modifications to the container configuration to simplify their execution.
+
+## Expected results
+
```
Comment (agheata) [inline _gsocproposals/2020/proposal_ALICEgridcontainers.md]: Please add the results you expect from the student project, considered as criteria of success

File: _gsocproposals/2020/proposal_ALICEgridcontainers.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+## Requirements
+
```
Comment (agheata) [inline _gsocproposals/2020/proposal_ALICEgridcontainers.md]: Requirements for skills are mandatory

File: _gsocproposals/2020/proposal_ALICEgridcontainers.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+## Mentors
+ * [Maksim Storetvedt](mailto:maksim.melnik.storetvedt@cern.ch) (primary)
+ * [Costin Grigoras](mailto:consting@cern.ch) (backup)
+
+## Links
```
Comment (agheata) [inline _gsocproposals/2020/proposal_ALICEgridcontainers.md]: Relevant links are mandatory, on top of self-explaining text (e.g. Project GitHub repository)

File: _gsocproposals/2020/proposal_ALICEtestj.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+ * Become familiar with the existing Java based setup scripts;
+ * Port the Java setup scripts to a language agnostic format;
+ * Develop a command line utility that unifies the individual scripts.
+
+## Expected results
+
+## Requirements
+
+## Mentors
+ * [Nikola Hardi](mailto:nhardi@cern.ch) (primary)
```
Comment (agheata) [inline _gsocproposals/2020/proposal_ALICEtestj.md]: Update mentors.md as well please

File: _gsocproposals/2020/proposal_ALICEtestj.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+ * [Costin Grigoras](mailto:consting@cern.ch) (backup)
+
+## Links
```
Comment (agheata) [inline _gsocproposals/2020/proposal_ALICEtestj.md]: Links section only once

File: _gsocproposals/2020/proposal_ALICEgridcontainers.md (hunk 1 above)
Comment (xvallspl) [reply _gsocproposals/2020/proposal_ALICEgridcontainers.md]: Double spacing in "must  meet"

File: _gsocproposals/2020/proposal_ALICEgridcontainers.md (hunk 2 above)
Comment (xvallspl) [reply _gsocproposals/2020/proposal_ALICEgridcontainers.md]: Maybe "pre-configured" or "preconfigured"

Comment (xvallspl) [reply _gsocproposals/2020/proposal_ALICEgridcontainers.md]: " to start containers, gather and analyze the output of the submission to gauge the readiness of the site to execute the containers against a list of known conditions" 

This enumeration is not clear. Maybe "start containers, **and** gather and analyze the output of the submission to gauge...", or "start containers **and** gather and analyze the output of the submission, to gauge...""? (note the different commas, could not write them in bold)

**Comment by Atlantic777** [comment]
Dear @agheata , @xvallspl thank you for the suggestions. The proposals are updated accordingly.

**Pull request merged**